pregred-gui/src/app/components/app-footer/AppFooter.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './AppFooter.css';
+import {
+  FacebookOutlined,
+  InstagramOutlined,
+  YoutubeOutlined,
+  TwitterOutlined
+} from '@ant-design/icons';
+
+
+
+
+function AppFooter() {
+    return(
+      <div className="footer-container">
+        <div className="f-text">
+          <ul className="dot">
+            <li>About Us: <a href="www.pregred.com"> www.pregred.com </a> </li>
+            <li>Contact Us: 0768660203</li>
+          </ul>
+          <div className="social">
+            <p >Social  follows</p>
```

Comment:
Social follows - what does this mean? focus on whether the content makes sense. 
moreover, are we going to link facebook, insta, linked in profiles here? if not remove them.